graylog2-server/src/main/java/org/graylog2/configuration/IndexSetsDefaultConfigurationFactory.java

```diff
@@ -46,7 +47,7 @@ public IndexSetsDefaultConfiguration create() {
                 .replicas(elasticsearchConfiguration.getReplicas())
                 .indexOptimizationDisabled(elasticsearchConfiguration.isDisableIndexOptimization())
                 .indexOptimizationMaxNumSegments(elasticsearchConfiguration.getIndexOptimizationMaxNumSegments())
-                .fieldTypeRefreshInterval(elasticsearchConfiguration.getIndexFieldTypePeriodicalFullRefreshInterval().toSeconds())
```

Comment:
It is a different setting. This one is being used in the periodical poller to determine when to do a full refresh of field types (which takes longer): https://github.com/Graylog2/graylog2-server/blob/master/graylog2-server/src/main/java/org/graylog2/indexer/fieldtypes/IndexFieldTypePollerPeriodical.java#L224

Right now there is no configuration setting for the default field types refresh. Now that it is generally configurable through the web interface, I do not see a problem with that. If we still want to support this, e.g. for better automation, we should introduce a setting for it. IMO this would be outside of the scope of this PR though.

Reply:
ah, missed the named injection. get's me every time :facepalm: